src/platform-includes/getting-started-install/react-native.mdx

```diff
@@ -63,3 +63,15 @@ For Android, we hook into Gradle for the source map build process. When you run
 If you have enabled Gradle's `org.gradle.configureondemand` feature, you'll need a clean build, or you'll need to disable this feature to upload the source map on every build.
 
 To disable this feature, set `org.gradle.configureondemand=false` or remove it as its default value is disabled, do this in the `gradle.properties` file.
+
+### Uninstall
```

Comment:
I'd not put it here, this is the install step and I'd not take their focus away.
TBH I am not sure where to place this, maybe @imatwawana has ideas? I guess no other SDK has it.

Reply:
I agree I haven't found any other uninstall steps so I wasn't sure where would be the best place.